spec/command-registry-spec.coffee

```diff
@@ -237,6 +241,61 @@ describe "CommandRegistry", ->
         {name: 'namespace:command-1', displayName: 'Namespace: Command 1'}
       ]
 
+    it "returns commands with manual displayNames if set in the listener", ->
```

Comment:
I almost think these two tests could be collapsed into one with a description like "returns arbitrary metadata from the command's registration". Testing that the name doesn't get overridden could just ride along with the main test with maybe a comment.